backend/router/misc.router.js

```diff
@@ -5,8 +5,7 @@ const {
   getUsersStatistics,
   getMoviesStatistics,
   getSeriesStatistics,
-  getPaymentStatistics,
-} = require("../controller/miscController.js");
+} = require("../controller/misc.controller.js");
 const authorizeRoles = require("../middleware/authorizeRoles");
```

Comment:
.js, please check all of these, either add them everywhere or remove them from everywhere.

Reply:
added .js